### PR TITLE
[one-cmds] Update one-prepare-venv packages

### PR DESCRIPTION
**Pull request by seanshpark**
This will update one-prepare-venv packages.
- related packages version are fixed to follow TF release date

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
for #11255 , from draft  #11278

File: compiler/one-cmds/one-prepare-venv [hunk 1]
```diff
@@ -79,7 +78,7 @@ if [[ ! -z "$ONE_PREPVENV_TORCH_STABLE" ]]; then
 fi
 # TODO remove torch message
 echo "Torch from '${ONE_PREPVENV_TORCH_STABLE}' -> '${TORCH_STABLE_URL}'"
-${VENV_PYTHON} -m pip ${PIP_OPTIONS} install torch==1.11.0+cpu -f ${TORCH_STABLE_URL}
+${VENV_PYTHON} -m pip ${PIP_OPTIONS} install torch==1.13.1+cpu -f ${TORCH_STABLE_URL}
```
Comment (jinevening) [inline compiler/one-cmds/one-prepare-venv:81]: Just curiosity. How torch version was decided? Is it a version near the TF release date?

**Review by jinevening — APPROVED** [review]
LGTM

File: compiler/one-cmds/one-prepare-venv (hunk 1 above)
Comment (seanshpark) [reply compiler/one-cmds/one-prepare-venv:81]: 1.13.1 is last of 1.x version.
Next is 2.0, March 16, 2023.
I think upgrading to 2.x would be left to next step.

https://pypi.org/project/torch/#history

**Pull request merged**